mapbox/vs30/scripts/basin_z_values/gen_sites_in_basin.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+import time
+import argparse
+
+from numba import jit, njit
+import numba
+import numpy as np
+import pandas as pd
+from Velocity_Model.basins import basin_outlines_dict
+from qcore import geo, formats
+import pygmt
+
+GRID_SPACING = 100
+BASIN_VER = "2.07"
+
+
+@jit(nopython=True)
+def is_inside_postgis(polygon, point):
```

Comment:
should these functions not have docstrings?

Reply:
Haha, I admit I was slack as this code is meant to be temporary and will be integrated into the new velocity model code. Yet I can see a point in doing things properly, as it will help the rewrite in the future. Sure, will re-submit.

Reply:
I think it would also be good to reference the source of this code, will try and find the stackoverflow question this comes from

Reply:
Actually this function exists in qcore, so should just be using that version?
https://github.com/ucgmsim/qcore/blob/5cfacc0354938232b984fe209197351031f18194/qcore/point_in_polygon.py

Reply:
yes, I mentioned this in this morning's statndup :)